### DISCUSSION TITLE
Is node reboot mandatory for completing a deployment?

### DISCUSSION THREAD

**Discussion by bqian20** [ostreedev/ostree#3325] [Q&A] [answered]
Would that be possible to deploy 2 deployments, without a reboot in between, and the deployments are correctly setup such that the 2nd one is on top of the first one?

**Comment by dbnicholson** [comment]
I don't think so. Somehow the new deployment needs to be mounted at the root path, and the only realistic way to do that is from the initramfs.

**Comment by cgwalters** [comment, marked as answer]
You're looking for https://github.com/ostreedev/ostree/issues/3242
